### PR TITLE
KHP3-6328 - Alignment of the Card Layout Header for Case Management Encounter Table

### PR DESCRIPTION
**Pull request by its-kios09**
…state

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

see:- https://thepalladiumgroup.atlassian.net/browse/KHP3-6328

## Screenshots
case-Management-encounter

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "zod": "^3.21.4"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^5.7.3-pre.2161",
```
Comment (donaldkibet) [inline package.json:34]: We should not be changing this version. We should only update `yarn.lock` file if we want to bump `@openmrs/esm-framework` version

File: package.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
     "jest-environment-jsdom": "^28.1.2",
     "lerna": "^4.0.0",
     "lodash": "^4.17.21",
-    "openmrs": "next",
+    "openmrs": "^5.7.3-pre.2161",
```
Comment (donaldkibet) [inline package.json:76]: Same as here

File: packages/esm-patient-clinical-view-app/src/case-management/encounters/case-encounter-header.scss [hunk 3]
```diff
@@ -3,31 +3,18 @@
 @use '@carbon/type';
 @import '~@openmrs/esm-styleguide/src/vars';
 
-.headerContainer {
-  display: flex;
-  justify-content: space-between;
+.widgetCard {
+  height: 50% !important;
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/case-management/encounters/case-encounter-header.scss:7]: Can we achieve the height without `!important` not recommended to use it

File: packages/esm-patient-clinical-view-app/src/case-management/encounters/case-encounter-header.scss [hunk 4]
```diff
@@ -3,31 +3,18 @@
...
-  border-bottom: 0.375rem solid var(--brand-03);
-  position: absolute;
-  left: 0;
-  bottom: -0.5rem;
-  top: 0.5rem;
+.controlsContainer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 1rem;
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/case-management/encounters/case-encounter-header.scss:15]:
```suggestion
  gap: layout.$spacing-05;
```

Comment (donaldkibet) [reply packages/esm-patient-clinical-view-app/src/case-management/encounters/case-encounter-header.scss:15]: Lets leverage carbon spacing tokens as we move forward

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (its-kios09) [reply package.json:34]: I will look at it

File: package.json (hunk 2 above)
Comment (its-kios09) [reply package.json:76]: same i will rectifying it

**Review by donaldkibet — APPROVED** [review]
Thanks @its-kios09

**Pull request merged**